asset/images/icons/invertibrates.svg

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Minor nitpick but I think the filename should be invertebrates not invertibrates?

Reply:
Yes, I'll change the name. Good catch!